build/Calamari.Consolidated.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <Target Name="ClearOldConsolidatedCalamariPackages" BeforeTargets="_CopyOutOfDateSourceItemsToOutputDirectory">
+    <ItemGroup>
+       <FilesToDelete Include="$(OutDir)\Calamari.*.zip"/>
+     </ItemGroup>
```

Comment:
Indentation not consistent :)

Reply:
I've found a better way that mitigates the drawback mentioned in the PR Description, so I'm going to fix this indentation at the same time as updating the approach. Just doing local testing now to double-check I haven't broken anything, then will push.